src/Abp.Zero.EntityFramework/EntityHistory/EntityHistoryHelper.cs

```diff
@@ -487,6 +509,33 @@ private void UpdateChangeSet(DbContext context, EntityChangeSet changeSet)
                 var entityType = GetEntityType(context.As<IObjectContextAdapter>().ObjectContext, entityEntry.GetEntityBaseType(), useClrType: false);
                 entityChange.EntityId = GetEntityId(entityEntry, entityType);
 
+                /* Update audit field */
+                var removePropertyChanges = new List<EntityPropertyChange>();
+                foreach (var propertyChange in entityChange.PropertyChanges.
```

Comment:
Can we use similar logic to the code here?
https://github.com/aspnetboilerplate/aspnetboilerplate/blob/df144d13800537d0e21734098306ee47dac848ec/src/Abp.Zero.EntityFramework/EntityHistory/EntityHistoryHelper.cs#L492-L504

Probably something like this.
```csharp
var properties = entityType.Properties;
foreach (var property in properties)
{
    var propertyEntry = entityEntry.Property(property.Name);
    var propertyChange = entityChange.PropertyChanges.FirstOrDefault(pc => pc.PropertyName == property.Name);

    //make sure test case cover post saving (for foreign key update)
    if (propertyChange == null)
    {
        if (propertyEntry.HasChanged())
        {
              // remove property change
              // or
              // update propertyChange.NewValue
        }
    }
}
```